tensorboard/webapp/runs/store/runs_selectors.ts

```diff
@@ -79,6 +79,25 @@ export const getRuns = createSelector(
   }
 );
 
+export const getRunsFromExperimentIds = (experimentIds: string[]) =>
+  createSelector(
+    getDataState,
+    (state: RunsDataState): Array<Run & {experimentId: string}> => {
+      return experimentIds.reduce((runs, experimentId) => {
+        const runsIdsForExperiment = (state.runIds[experimentId] || []).filter(
+          (id) => Boolean(state.runMetadata[id])
+        );
+        runsIdsForExperiment.forEach((runId) => {
+          runs.push({
+            ...state.runMetadata[runId],
+            experimentId,
+          });
+        });
```

Comment:
I would prefer something like this:
```
(state.runsIds[experimentId] || []).forEach(runId => {
  if(state.runMetadata[id]) {
    runs.push({...state.runMetadata[runId], exprimentId});
  }
})
```
inside the reduce function.

Reply:
I've updated the formatting to meet someplace in between